src/content/glossary/index.md

```diff
@@ -424,6 +430,18 @@ A procedural (imperative) programming language with syntax that is similar to Ja
 
 A [hard fork](#hard-fork) of the Ethereum blockchain, which occurred at block 2,675,000 to address more denial-of-service attack vectors and clear state (see [Tangerine Whistle](#tangerine-whistle)). Also, a replay attack protection mechanism.
 
+### stablecoin {#stablecoin}
+
+An [ERC-20 token](#token-standard), backed by a reserve asset, with price pegged to real-world asset value. Examples of uses include pegging an ERC-20 tokens value to the value of fiat currency, precious metals, or other cryptocurrencies.
+
+<a href="/en/eth/#tokens">More about tokens<Emoji text=":point_right:" size={1} marginLeft={0.75} /></a>
+
+### staking {#staking}
+
+The act of locking up a quantity of [ether](#ether) (your stake in the network) to earn the right to validate [transactions](#transaction) and propose [blocks](#block) under a [proof-of-stake](#pos) consensus model. For [Ethereum 2.0](#serenity), each validator requires staking 32 ether by depositing to a [smart contract](#smart-contract), which enables each validator to attest to transactions and propose blocks. Being a validator carries responsibilities, and comes with rewards for fulfilling them, and varying degrees of penalties for not.
```

Comment:
I think we should capitalise "Ether". 

Also, I think this definition could be simplified somewhat – not sure we need to mention the smart contract or attestations. Also, we'll have dedicated Eth2 and staking pages to link users to soon, for more info.

A suggested edit:

"Depositing a quantity of Ether (your stake) to become a validator and secure the network. A validator checks transactions and proposes blocks under a proof-of-stake consensus model. Staking gives you an economic incentive to act in the best interests of the network. You'll get rewards for carrying out your validator duties, but lose varying amounts of ETH if you don't."

Reply:
👍 That sounds good to me, especially as long as we can link to more info.

The note about 'ether' being capitalized I would honestly argue against though. Per our own white paper:
![image](https://user-images.githubusercontent.com/54227730/98011474-bc8efd00-1dac-11eb-9d66-4bd5a8acac5a.png)
Just like 'dollars' or 'bitcoin', I always use 'ether' (or all capital tickers 'USD' 'BTC' 'ETH') when referring to the currency personally. 

That being said, searching the repo for each spelling I see a _lot of both_. So, at the very least I do think it's worth clarifying this and maybe making an issue to standardize across the site.

Reply:
Ok, let's go for `ether` and `ETH` – will open a separate PR to address this!